testing/framework/TestCmd.py

```diff
@@ -1590,7 +1590,10 @@ def stdout(self, run=None):
         elif run < 0:
             run = len(self._stdout) + run
         run = run - 1
-        return self._stdout[run]
+        try:
+            return self._stdout[run]
+        except IndexError:
```

Comment:
When would this happen? Can you add a comment?

Reply:
Had an exception... I guess there's no contents to self._stdout at all, so can't index into it.
```
493/1179 (41.82%) C:\\Python37\\python.exe -tt test\Interactive\implicit-VariantDir.py
timed out waiting for C:\Users\mats\AppData\Local\Temp\testcmd.6464.8hocmiyx\1 to exist

Traceback (most recent call last):
  File "test\Interactive\implicit-VariantDir.py", line 106, in <module>
    test.wait_for(test.workpath('1'))
  File "C:\users\mats\Documents\github\scons\testing\framework\TestSCons.py", line 1375, in wait_for
    stdout = self.stdout()
  File "C:\users\mats\Documents\github\scons\testing\framework\TestCmd.py", line 1593, in stdout
    return self._stdout[run]
IndexError: list index out of range
```

Reply:
does your change skip the exception but fail the test if it should fail?

Reply:
hmmm.... doesn't look like there's a case for what happens. not sure what should happen.

Reply:
now that it's merged... `implicit-VariantDir` passes, which maybe it shouldn't?  The two from the other test, `sconsign/script/Configure.py` and `dblite.py` (same dir) both pass now.